src/UI/Art/SpriteArtist/PlayerSetupCommanderArtist.java

```diff
@@ -26,9 +30,14 @@ public class PlayerSetupCommanderArtist
 
   private static SlidingValue tagPickerOffsetX = new SlidingValue(0);
 
-  static final BufferedImage qtip = SpriteUIUtils.makeTextFrame("Q: Kick CO", 3, 2);
+  static final BufferedImage kickTip = SpriteUIUtils.makeTextFrame("Q: Kick CO", 3, 2);
+  static final BufferedImage toGameTip = SpriteUIUtils.makeTextFrame("Q: By Game", 3, 2);
+  static final BufferedImage toArmyTip = SpriteUIUtils.makeTextFrame("Q: By Army", 3, 2);
   static final BufferedImage etip = SpriteUIUtils.makeTextFrame("E: CO Info", 3, 2);
 
+  static final int textWidth = SpriteLibrary.getLettersSmallCaps().getFrame(0).getWidth();
+  static final int textHeight = SpriteLibrary.getLettersSmallCaps().getFrame(0).getHeight();
```

Comment:
We should make a less painful way to get this info. Also it hurts me a little bit that we handle the standard vs small-caps fonts in _completely_ different ways.

Reply:
Kerning isn't terribly important with a monospace font, to be fair.

Reply:
Obviously out of scope for this MR, I'm just complaining because I do that.